app/models/block.rb

```diff
@@ -121,8 +118,11 @@ def invalid!
 #  dao                        :string
 #  primary_reward             :decimal(30, )    default(0)
 #  secondary_reward           :decimal(30, )    default(0)
-#  nonce                      :string
 #  chain_root                 :binary
+#  difficulty                 :decimal(30, )    default(0)
```

Comment:
`difficulty` is an U256, larger than `decimal(30)`

Reply:
thanks, I will fix this.